pkg/handlers/v1/scanasync.go

```diff
@@ -128,6 +132,12 @@ func (h *ScanAsyncSubmit) Handle(ctx context.Context, in ScanInput) (interface{}
 		ScanInput:  in,
 		Identifier: id,
 	}
+	if ain.Scripts == nil {
+		ain.Scripts = make([]string, 0)
```

Comment:
Q: what's the advantage of zero size allocation vs keeping it nil? Apart from making things like length() work?

Reply:
`len()` actually works just fine with a `nil` slice. The purpose of doing this is to ensure the value is non-`nil` so it marshals to an array instead of a `null`.